src/UiCalendarControls.vue

```diff
@@ -92,11 +92,13 @@ export default {
         nextMonthDisabled() {
             const firstDayOfNextMonth = dateUtils.clone(this.dateInView);
 
+            const sortedYearRange = this.yearRange.sort((a, b) => a - b);
```

Comment:
Make this a computed property `yearRangeSorted` so the sorting is only done once, with the results cached until `yearRange` is changed.

Reply:
I saw that you cloned it to to avoid mutating the prop 👍. Could you move it to a computed property as well?